config/database.yml

```
@@ -1,7 +1,7 @@
 default: &default
   adapter: postgresql
   encoding: unicode
-  pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
+  pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 }.to_i + ENV.fetch("GOOD_JOB_MAX_THREADS", 4).to_i %>
```

It might not really matter, just made me do a double-take that there are two different ways of calling the fetch... or do they need to be different?
```suggestion
  pool: <%= ENV.fetch("RAILS_MAX_THREADS", 5).to_i + ENV.fetch("GOOD_JOB_MAX_THREADS", 4).to_i %>
```

---

You're right, the `5` should be the second parameter. I've noticed this before in here, but never bothered to fix. Now's a good time to do that.